tests/fixers/test_admin_register.py

```diff
@@ -470,6 +454,80 @@ class AuthorAdmin(CustomModelAdmin):
     )
 
 
+def test_multiple_model_multiline_registration():
```

Comment:
let's add a duplicate of this test where the models are originally registered in the other order, to check the sorting behaviour